application/common/mail/mfa-rate-limit.html.php

```diff
@@ -12,6 +12,16 @@
  * @var string $locked
  * @var string $lastChangedUtc
  * @var string $lastSyncedUtc
+ * @var string $lastLoginUtc
+ * @var string $passwordExpiresUtc
+ * @var string $emailSignature
+ * @var string $helpCenterUrl
+ * @var string $idpDisplayName
+ * @var string $passwordForgotUrl
+ * @var string $passwordProfileUrl
+ * @var string $supportEmail
+ * @var string $supportName
+ * @var bool   $isMfaEnabled
  */
 ?>
 <p>
```

Comment:
We still don't have the MFA-rate-limit email content in place.

Reply:
oh right, I'll change default value of whether we should send it to false in each IdP TF vars config

Reply:
Okay. We could also probably add it fairly quickly (There have been several failed attempts at 2-Step Verification on your account. If that was not you, you should change your password immediately. If you have any questions, please contact {supportName} at {supportEmail.}). I'm fine with either option.

Reply:
already done